### DISCUSSION TITLE
Select group instead of mesh?

### DISCUSSION THREAD

**Discussion by sajal353** [pmndrs/drei#800] [Q&A]
Hello,

I'm trying to use Transform controls to move a gltf model which is inside <Select>. But instead of the whole model, it selects individual meshes. 

Please help me out.

**Comment by mwmwmw** [comment]
It should just be a matter of passing the group to `<TransformControls />`

`<TransportControl object={mygroup} />`

Here is an example, I set click events on various groups. When a group is clicked, I set it as the selected object, and then pass the selected object in `TransformControls`. You can then take this engine apart and interact with each piece. 

https://codesandbox.io/s/transformcontrols-on-gltf-vget1s?file=/src/2CylinderEngine.js
